haystack/document_stores/faiss.py

```diff
@@ -276,6 +275,14 @@ def write_documents(
             ) as progress_bar:
                 for i in range(0, len(document_objects), batch_size):
                     if add_vectors:
+                        if not self.faiss_indexes[index].is_trained:
+                            raise ValueError(
+                                "FAISS index of type {} must be trained before adding vectors. Call `train_index()` "
+                                "method before adding the vectors. For details, refer to the documentation: "
+                                "https://docs.haystack.deepset.ai/reference/document-store-api#faissdocumentstoretrain_"
```

Comment:
the link seems cut off, it should be: https://docs.haystack.deepset.ai/reference/document-store-api#faissdocumentstoretrain_index

also, can we make this link accessible:
```suggestion
                                "[FAISSDocumentStore API](https://docs.haystack.deepset.ai/reference/document-store-api#faissdocumentstoretrain_index)."
```

Reply:
Done. Thanks a lot @agnieszka-m :)